src/events/messageCreate.js

```diff
@@ -1,15 +1,10 @@
 const Discord = require('discord.js');
 const { button: disableNLPButton } = require('../buttons/disable-nlp');
-const wiiuSupportCodes = require('../console-errors/wiiu/support-codes');
-const threeDSSupportCodes = require('../console-errors/3ds/support-codes');
-const pretendoSupportCodes = require('../console-errors/pretendo/support-codes');
+const { button: expandErrorButton } = require('../buttons/expand-error');
+const checkForErrorCode = require('../utils/errorCode').checkForErrorCode;
```

Comment:
For consistencies sake this should be written  as `const { checkForErrorCode } = require('../utils/errorCode');`